src/Sylius/Bundle/ApiBundle/Resources/config/services/state_processors.xml

```diff
@@ -30,6 +30,12 @@
             <tag name="api_platform.state_processor" />
         </service>
 
+        <service id="sylius_api.state_processor.delete.locale" class="Sylius\Bundle\ApiBundle\StateProcessor\Delete\LocaleProcessor">
```

Comment:
The naming convention here is a mess, completely disregarding the FQCN at the top, we have `sylius_api.state_processor.shop.channel` which doesn't indicate when it's used and uses remove and persist processors inside, as well as `sylius_api.state_processor.post.upload_avatar_image` which is very specific.

Reply:
That's true, the naming convention in this file is indeed a mess, but it is not connected with this PR and we should clean it up in a separate one. I believe that the newly added `Delete/LocaleProcessor` is quite a good example to make other consistent to each other, wdyt?

Reply:
Not necessarily, we have a few that work on multiple actions, should we split the implementation per action in that case or extract another syntax?

Reply:
I'm resolving after the internal discussion, we definitely need to make these processors consistent with each other, but in a separate PR 🖖🏻